superset-frontend/src/components/DvtTable/index.tsx

```diff
@@ -33,25 +33,28 @@ import {
   StyledTableTitle,
   StyledTableIcon,
   StyledTableCheckbox,
+  StyledTableUrl,
 } from './dvt-table.module';
 import DvtPopper from '../DvtPopper';
 
 interface HeaderProps {
   id: number;
   title: string;
   field?: string;
-  folderIcon?: boolean;
-  heartIcon?: boolean;
-  onLink?: boolean;
+  icon?: string;
+  iconActive?: string;
+  iconClick?: () => {};
+  url?: string;
```

Comment:
url?: string; aslında urlField olmalı.

urlField api objectname yazılacak yazısı mavi ve link için click içinde history.push(item.link gibi olacak)

url: "/" yazmışsın bu değil örneğimi apiData {id: 1, link: "/dashboard/list/94"} sen yeni data format için link koyarsın mesala  urlField: "link" bu yakalarsa link çalışması gerekiyor.